### PR TITLE
upgrade dependencies

### PR DESCRIPTION
**Pull request by etarancon**

### CODE REVIEW

File: tests/dummy/app/router.js [hunk 1]
```diff
@@ -2,10 +2,10 @@ import Ember from 'ember';
 import config from './config/environment';
 
 const Router = Ember.Router.extend({
-  location: config.locationType
+	location: config.locationType
 });
 
-Router.map(function() {
+Router.map(() => {
```
Comment (josex2r) [inline tests/dummy/app/router.js]: this shouldn't be an arrow function

**Review by adrigzr — APPROVED** [review]

File: tests/unit/amd-test.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ test('it requires a module with its value and undefines it', (assert) => {
 	amd.define('foo', value);
 
 	assert.equal(amd.require('foo'), value);
-	assert.notOk(requirejs.entries['foo']);
+	assert.notOk(requirejs.entriesfoo);
```
Comment (adrigzr) [inline tests/unit/amd-test.js]: This is wrong

**Review by adrigzr — CHANGES_REQUESTED** [review]

**Review by adrigzr — DISMISSED** [review]

File: tests/dummy/app/router.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 import config from './config/environment';
 
 const Router = Ember.Router.extend({
```
Comment (shokmaster) [inline tests/dummy/app/router.js]: `import EmberRouter from '@ember/routing/router';`

File: tests/helpers/destroy-app.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import Ember from 'ember';
 
 export default function destroyApp(application) {
```
Comment (shokmaster) [inline tests/helpers/destroy-app.js]: `import { run } from '@ember/runloop';`

Comment (etarancon) [reply tests/helpers/destroy-app.js]: done

File: tests/dummy/app/router.js (hunk 3 above)
Comment (etarancon) [reply tests/dummy/app/router.js]: done

**Review by adrigzr — APPROVED** [review]

**Pull request merged**